tests/test_parameterspace.py

```diff
@@ -512,9 +512,9 @@ def test_to_from_dict(num_samples=128):
         )
 
         for k in sample.keys():
-            assert sample[k] == s1[k] or math.isclose(sample[k], s1[k], rel_tol=1e-15)
-            assert sample[k] == s2[k] or math.isclose(sample[k], s2[k], rel_tol=1e-15)
-            assert sample[k] == s3[k] or math.isclose(sample[k], s3[k], rel_tol=1e-15)
+            assert sample[k] == s1[k] or math.isclose(sample[k], s1[k], rel_tol=3e-15)
+            assert sample[k] == s2[k] or math.isclose(sample[k], s2[k], rel_tol=3e-15)
+            assert sample[k] == s3[k] or math.isclose(sample[k], s3[k], rel_tol=3e-15)
```

Comment:
I'm wondering if and why we still need the quality condition. I would expect the equality condition is contained in the isclose check. What am I missing?
Since we are sampling in the interval `[0, 1]` I'd expect the default `rel_tol=1e-09` to be fine as well, without the risk of having real errors slip through. @sfalkner, what do you think?

Reply:
Valid question, I wondered the same thing and tried it. ***But***: Turns out we have categorical parameters to compare in our test case, too (L.481), and `isclose()` won't handle strings ;-)

Reply:
Regarding the `rel_to`: I would prefer the default of `1e-09`, too. Waiting for @sfalkner response, if this is close enough.

Reply:
Aha, I mistakenly assumed that we are comparing the numerical representation. If we are comparing the actual samples, this makes total sense then; and for the precision is even more reason in my view to go with the more relaxed but likely still sufficient default.

Reply:
After discussion with @LGro, I did left out `rel_to` for the default.